webpack.dev.js

```diff
@@ -5,31 +5,35 @@ const HTMLPlugin = require('html-webpack-plugin');
 const HTMLTemplate = require('html-webpack-template');
 const common = require('./webpack.common.js');
 
-module.exports = env => merge(common(env), {
-  plugins: [
-    new webpack.HotModuleReplacementPlugin(),
-    new HTMLPlugin({
-      inject: false,
-      template: HTMLTemplate,
-      title: 'webpack',
-      appMountId: 'root',
-      mobile: true,
-      favicon: './favicon.ico',
-      meta: {
-        description: '...'
-      }
-    }),
-    new webpack.optimize.CommonsChunkPlugin({
-      name: 'vendor',
-      chunks: [ 'index' ]
-    })
-  ],
-  devServer: {
-    contentBase: path.resolve(__dirname, './dist'),
-    port: 3000,
-    hot: true,
-    inline: true,
-    compress: true,
-    historyApiFallback: true
-  }
-})
+module.exports = env =>
+  merge(common(env), {
+    plugins: [
+      new webpack.HotModuleReplacementPlugin(),
+      new HTMLPlugin({
+        inject: false,
+        template: HTMLTemplate,
+        title: 'webpack',
+        appMountId: 'root',
+        mobile: true,
+        favicon: './favicon.ico',
+        meta: {
+          description: '...'
+        }
+      }),
+      new webpack.optimize.CommonsChunkPlugin({
+        name: 'vendor',
+        chunks: ['index']
+      }),
+      new webpack.DefinePlugin({
+        'process.env.NODE_ENV': JSON.stringify('development')
+      })
+    ],
+    devServer: {
+      contentBase: path.resolve(__dirname, './dist'),
+      port: 3000,
+      hot: true,
+      inline: true,
+      compress: true,
+      historyApiFallback: true
+    }
+  });
```

Comment:
Same here, I'm not exactly sure what even changed. Was this intentional?

Reply:
Ah you have a sublime plugin for it or something, I see now. Yeah if you don't mind, let's wait to make all those changes as a separate commit until after this is done and merged. Then we can integrate it into the build process in a way that hits everyone's builds.